generator.go

```diff
@@ -28,7 +28,7 @@ func NewGenerator(structs []StructDef) generator {
 func (g generator) Generate(path, outputFile string, needCheck bool) error {
 	pkg := filepath.Base(path)
 	if pkg == "." {
-		pkg = "main"
+		pkg = "api"
```

Comment:
I think we should create and think about `genval` as some standalone tool for generating validations and don\`t use our internal packages and directories